validator/db/iface/interface.go

```diff
@@ -35,10 +35,10 @@ type ValidatorDB interface {
 	// Attester protection related methods.
 	AttestationHistoryForPubKeys(ctx context.Context, publicKeys [][48]byte) (map[[48]byte]*slashpb.AttestationHistory, error)
 	SaveAttestationHistoryForPubKeys(ctx context.Context, historyByPubKey map[[48]byte]*slashpb.AttestationHistory) error
-	HighestSignedTargetEpoch(ctx context.Context, publicKey [48]byte) (uint64, error)
```

Comment:
Just to confirm , this method has been there this whole time ? `Highest` instead of `Lowest`

Reply:
Ignore the comment